scenarios/kubernetes_verify.py

```diff
@@ -27,12 +27,12 @@
 
 
 VERSION_TAG = {
-    '1.10': '1.10-v20180221-3655e24ae',
-    '1.11': '1.11-v20190125-cc5d6ecff3',
-    '1.12': '1.12-v20190125-cc5d6ecff3',
-    '1.13': '1.13-v20190125-cc5d6ecff3',
+    '1.11': '1.11-v20190207-b40871ffa',
+    '1.12': '1.12-v20190207-b40871ffa',
+    '1.13': '1.13-v20190207-b40871ffa',
+    '1.14': '1.14-v20190207-b40871ffa',
     # this is master, feature branches...
-    'default': '1.13-v20190125-cc5d6ecff3',
+    'default': '1.14-v20190207-b40871ffa',
```

Comment:
we've migrated the verify & integration job to podutils for master - I think it should work for 1.14 as well :-)

Reply:
done